example_profile.mobileconfig

```diff
@@ -0,0 +1,87 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>PayloadContent</key>
+	<array>
+		<dict>
+			<key>PayloadDisplayName</key>
+			<string>Nudge Configuration</string>
+			<key>PayloadIdentifier</key>
+			<string>com.macadmins.nudge</string>
+			<key>PayloadType</key>
+			<string>com.macadmins.nudge</string>
+			<key>PayloadUUID</key>
+			<string>41AF5F92-8C0F-40B2-B6A6-09899F3D23A2</string>
+			<key>PayloadVersion</key>
+			<integer>1</integer>
+			<key>button_title_text</key>
+			<string>Ready to start the update?</string>
+			<key>button_sub_titletext</key>
+			<string>Click on the button below.</string>
+			<key>cut_off_date</key>
+			<string>2020-09-24-16:00</string>
+			<key>cut_off_date_warning</key>
+			<integer>14</integer>
+			<key>time_between_notifications</key>
+			<integer>0</integer>
+			<key>dismissal_count_threshold</key>
+			<integer>10</integer>
+			<key>logo_path</key>
+			<string>/Library/nudge/Resources/company_logo.png</string>
+			<key>main_subtitle_text</key>
+			<string>A friendly reminder from your local IT team</string>
+			<key>main_title_text</key>
+			<string>macOS Update</string>
+			<key>minimum_os_version</key>
+			<string>10.15.6</string>
+			<key>minimum_os_sub_build_version</key>
+			<string>19G2021</string>
+			<key>more_info_url</key>
+			<string>https://www.google.com</string>
+			<key>no_timer</key>
+			<false/>
+			<key>paragraph1_text</key>
+			<string>A fully up-to-date device is required to ensure that IT can your accurately protect your computer.</string>
+			<key>paragraph2_text</key>
+			<string>If you do not update your computer, you may lose access to some items necessary for your day-to-day tasks.</string>
+			<key>paragraph3_text</key>
+			<string>To begin the update, simply click on the button below and follow the provided steps.</string>
+			<key>paragraph_title_text</key>
+			<string>A security update is required on your machine.</string>
+			<key>path_to_app</key>
+			<string>/Applications/Install macOS Mojave.app</string>
+			<key>random_delay</key>
+			<false/>
+			<key>screenshot_path</key>
+			<string>/Library/nudge/Resources/update_ss.png</string>
+			<key>timer_day_1</key>
+			<integer>600</integer>
+			<key>timer_day_3</key>
+			<integer>7200</integer>
+			<key>timer_elapsed</key>
+			<integer>10</integer>
+			<key>timer_final</key>
+			<integer>60</integer>
+			<key>timer_initial</key>
+			<integer>14400</integer>
+			<key>update_minor</key>
+			<false/>
+			<key>update_minor_days</key>
+			<integer>14</integer>
+		</dict>
+	</array>
+	<key>PayloadDisplayName</key>
+	<string>Nudge Configuration</string>
+	<key>PayloadIdentifier</key>
+	<string>com.macadmins.nudge.2539D674-2B6B-4553-B2F5-2AEBC90CDD7C</string>
```

Comment:
take off the uuid here. not needed and will be easier to remove when people are playing around.